cpu/mips32r2_common/include/periph_cpu.h

```diff
@@ -0,0 +1,3 @@
+/* This file must exist to get timer code to build */
```

Comment:
hm, as there are `periph_cpu.h` files in the specific cpu folders, there should be no need for this file here? Seems like something is not quite in order if this file is really needed...